src/operator/src/insert.rs

```diff
@@ -66,6 +66,22 @@ pub struct Inserter {
 
 pub type InserterRef = Arc<Inserter>;
 
+enum AutoCreateTableType {
+    Logical(String),
+    Physical,
+    Log,
+}
```

Comment:
These are not concepts at same level. Do we really need `Log` table type here?

Reply:
I don't think we need a `Log` table type. cc @fengjiachun

Reply:
Here is just the creation statement used to differentiate between different types of tables. The log table is also a normal table, just with append_mode turned on.